polling_stations/apps/file_uploads/management/commands/check_pending.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+from django.conf import settings
+from django.utils.timezone import now
+from file_uploads.models import Upload
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        from_time = now() - timedelta(minutes=5)
+        qs = Upload.objects.filter(timestamp__lte=from_time).filter(
+            warning_about_pending_sent=False
+        )
+
+        if len(qs) == 0:
+            print("No failed uploads")
+        else:
+            qs.update(warning_about_pending_sent=True)
```

Comment:
I think this needs a little bit of rework, depending on how you plan to deal with Will's comment. 

## Options 1

If we are going to send a single email for _all_ uploads in the qs, then we should do something like:

```
qs = Upload.objects.stuck_in_pending()
if not qs:
    self.stdout.write("No uploads found")
    return

[send email]
```

# Option 2

To send one email per upload in the qs:

```
qs = Upload.objects.stuck_in_pending()
for upload in qs:
    [email]
else:
    self.stdout.write("No uploads found")
```

(Note `self.stdout.write` rather than `print` as it's better for testing)


However, I'm not sure we need to print that no uploads are found, as that will be the common case and don't think we need that filling up our logs.